test/appium/views/profile_view.py

```diff
@@ -374,6 +374,24 @@ def __init__(self, driver):
         self.locator = self.Locator.accessibility_id('mailserver-connect-button')
 
 
+class ActiveNetworkName(BaseText):
+
+    def __init__(self, driver):
+        super(ActiveNetworkName, self).__init__(driver)
+        self.locator = self.Locator.text_part_selector('WITH UPSTREAM RPC')
+
+
+class AboutButton(BaseButton):
+    def __init__(self, driver):
+        super(AboutButton, self).__init__(driver)
+        self.locator = self.Locator.xpath_selector(
+            '//android.view.ViewGroup[@content-desc="about-button"]/android.view.ViewGroup')
```

Comment:
It's better to use accessibility ids where it's possible, here it's 'about-button'